syncano_cli/sync/utils.py

```diff
@@ -0,0 +1,18 @@
+# coding=UTF8
+from __future__ import print_function, unicode_literals
+
+
+def compare_dicts(new, old):
```

Comment:
I gave you whole code for this function in PR comment ? Why haven't you used it ?
So again
```
def diff_dicts(old, new):
    old_keys = set(old or ())
    new_keys = set(new or ())
    common = new_keys & old_keys
    added = new_keys - old_keys
    removed = old_keys - new_keys
    same = set(k for k in common if old[k] == new[k])
    return same, added, removed, common - same
```